File: AdotePetWeb/backend/api/model/PedidoAdocao.php
<?php

/**
 * @author Thaís
 */
class PedidoAdocao {
   
    private $dataHora;
    private $descricao;
    private $situacao;
    private $dataAlteracaoSituacao;
   
    public function get_dataHora() {
        return $this->dataHora;
    }

    public function get_descricao() {
        return $this->descricao;
    }

    public function get_situacao() {
        return $this->situacao;
    }
    
    public function get_dataAlteracaoSituacao() {
        return $this->dataAlteracaoSituacao;
    }

    public function set_dataHora($dataHora) {
        $this->dataHora = $dataHora;
        return $this;
    }
    
    public function set_descricao($descricao) {
        $this->descricao = $descricao;
        return $this;
    }

    public function set_situacao($situacao) {
        $this->situacao = $situacao;
        return $this;
    }
    
    public function set_dataAlteracaoSituacao($dataAlteracaoSituacao) {
        $this->dataAlteracaoSituacao = $dataAlteracaoSituacao;
        return $this;
    }

    

}

File: AdotePetJavaProjetct/src/adotepet/PedidoAdocao.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package adotepet;

import java.util.Date;

/**
 *
 * @author Matheus
 */
public class PedidoAdocao {
    public PedidoAdocao(Date dataHora, String descricao,
                        Adotante adotante) {
        this.dataHora = dataHora;
        this.descricao = descricao;
        this.situacao = "EM ABERTO";
        this.adotante = adotante;
    }

    private Date dataHora;
    private String descricao;
    private String situacao;
    private Adotante adotante;

    public Date getDataHora() {
        return dataHora;
    }

    public void setDataHora(Date dataHora) {
        this.dataHora = dataHora;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }

    public String getSituacao() {
        return situacao;
    }

    public void setSituacao(String situacao) {
        this.situacao = situacao;
    }

    public Adotante getAdotante() {
        return adotante;
    }

    public void setAdotante(Adotante adotante) {
        this.adotante = adotante;
    }
}
